modules/tinymce/src/themes/silver/test/ts/browser/editor/SilverRenderTest.ts

```diff
@@ -25,7 +25,7 @@ describe('browser.tinymce.themes.silver.editor.SilverRenderTest', () => {
   const hook = TinyHooks.bddSetupLight<Editor>({
     plugins: 'test',
     base_url: '/project/tinymce/js/tinymce',
-    setup: (editor) => {
+    setup: (editor: Editor) => {
```

Comment:
Just curious, why is the type removed for some setups, and added for others?

Reply:
It's removed in demos because we have the `TinyMCE` type that it can infer the editor from. For McAgar, it needs to work with TinyMCE 4, 5, 6, etc... so it unfortunately doesn't have the type information available for the options. Now we could change that, if we passed through another generic like we do for `Editor` (which is the return type), however that's a bigger change I'd prefer to do separately if we wanted to go down that path.

Reply:
Interesting, thanks @lnewson